docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md

```diff
@@ -7,99 +7,100 @@ description: >
   This section explains how the United Manufacturing Hub is used practically 
 ---
 
-## 1. Extract data using factorycube-edge
+## 0. Video: From sensor to dashboard
 
-**The basic approach for data processing on the local hardware is to extract data from various data sources (OPC/UA, MQTT, Rest), extract the important information, and then make it available to the United Manufacturing Hub via a predefined interface (MQTT). For this data processing on the local hardware we use Nodered.**
+TODO
+#455
 
-To extract and pre-process the data from different data sources we use the open source software node-red. node-red is a low-code programming for event-driven applications.
+## 1. Introduction
 
-If you haven't worked with node-red yet, [here](https://nodered.org/docs/user-guide/) is a good documentation from node-red!
+UMH Hub has a wide range of ports and connectors. 
 
-{{< imgproc nodered Fit "800x800" >}}{{< /imgproc >}}
+{{< imgproc logos_data_sources Fit "800x800" >}}{{< /imgproc >}}
```

Comment:
This cross out. And the subline "could contain sensitive information".
It is best to use "www.draw.io" for the graphic.
Insert the Weihenstephaner standard logo and the logos from our website in a "chaotic order".

![WS](https://user-images.githubusercontent.com/77407527/144581210-7f3b051e-1f43-4633-bc9a-01f4e03f52b6.png)
<img width="249" alt="image" src="https://user-images.githubusercontent.com/77407527/144581303-c22d98a5-12cc-46ee-9574-a5bdffaf1889.png">